# true  if laser_frame
# false if camera_frame
bool laser_frame

# the angular difference (in radians)
# between the individual pixels in the image
# in the x-direction
float64 ang_res_x

# the angular difference (in radians)
# between the individual pixels in the image
# in the y-direction
float64 ang_res_y

# the angle (in radians) defining the horizontal
# bounds of the sensor
float64 max_ang_w

# the angle (in radians) defining the vertical
# bounds of the sensor
float64 max_ang_h

# tocomment
float64 offset_x
float64 offset_y

# tocomment
float64 min_range
float64 max_range
